frontend/src/views/Blacksmith.vue

```diff
@@ -311,8 +325,17 @@ export default {
       });
     },
 
-    viewNewWeapons(newWeaponCount = 1){
-      this.newForged = this.ownedWeaponIds.slice(-newWeaponCount);
+    viewNewWeapons(){
+      this.newForged = [];
+      this.ownedWeaponIds.forEach(x => {
+        this.newForged.push(x);
+      });
+      if(this.x1Forge === true){
```

Comment:
never check against true, just check if(this.x1Forge). perhaps instead, do `this.forgeAmount = 1 or 10` and check that. but really, this should be a number and you should do the math based on that number.